test/inspector/test-off-with-session-then-on.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper.js');
+
+function testResume(session) {
+  session.sendCommandsAndExpectClose([
+    { 'method': 'Runtime.runIfWaitingForDebugger' }
+  ]).expectStderrOutput('Debugger listening on');
+}
+
+function testDisconnectSession(harness) {
+  harness
+    .runFrontendSession([
+      testResume,
+    ]).expectShutDown(42);
+}
+
+const script = 'process._debugEnd();' +
+               'process._debugProcess(process.pid);' +
+               'process._debugEnd();' +
```

Comment:
I am testing if this fixes the failures. Server is starting (and printing out the message) on a background thread so I suspect the reason test fails is because process shuts down too fast. _debugEnd synchronizes the threads.